### DISCUSSION TITLE
Thanks for your work!

### DISCUSSION THREAD

**Discussion by lsoft** [zompinc/sync-method-generator#37] [Show and tell]
Hello @virzak! I cannot open discussion here, so I opened this issue.

I want to say thank you again for your work! Our big and old enterprise system benefit a lot from your ISG.

Your work inspired me to create an [extension for Visual Studio](https://marketplace.visualstudio.com/items?itemName=lsoft.SyncToAsyncExtension) that makes it easier to move from a synchronous method to an asynchronous one and back again:

https://raw.githubusercontent.com/lsoft/SyncToAsyncExtension/main/example0.png

I will be glad if you find this extension useful for yourself.

Thanks a lot! 👍

**Comment by virzak** [comment]
Amazing @lsoft!

Your extension helps me avoid 
```c#
_ = nameof(SyncVersionOfFunctionalityX);
```
in the testing code for this library as well as other projects.

I'll add it to the README.

Just out of curiosity, how many instances of the `[CreateSyncVersion]` do you have in your project and how long is the longest method which has it?

Cheers,
Victor

**Reply by lsoft** [reply]
@virzak we have 69 instances `CreateSyncVersion` now, max line count is about 100, we tried not to have loooong methods :)

But I did not covered all the places I want with `CreateSyncVersion`, the instance count will getting bigger.

**Reply by virzak** [reply]
That makes me extremely happy. The most I have across my projects is 20 at this time :)
